Swiftfin/Views/LibraryView/LibraryView.swift

```diff
@@ -71,15 +93,35 @@ struct LibraryView: View {
             } else if viewModel.items.isEmpty {
                 noResultsView
             } else {
-                libraryItemsView
+                switch libraryViewType {
+                case .grid:
+                    libraryGridView
+                case .list:
+                    libraryListView
+                }
             }
         }
         .navigationBarTitleDisplayMode(.inline)
         .toolbar {
             ToolbarItemGroup(placement: .navigationBarTrailing) {
+                Button {
+                    switch libraryViewType {
+                    case .grid:
+                        libraryViewType = .list
+                    case .list:
+                        libraryViewType = .grid
```

Comment:
It's a small thing, would it be weird if we put an `withAnimation` in here?

Reply:
Good recommendation! I tried it out but I think that it's better to just transition quickly. I don't expect people to be playing with this toggle.